types/ext/object-property-accessor.d.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2023  Yomitan Authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+export type ParsePathStringState = 'empty' | // Empty
```

Comment:
Don't use alignment indent, standard 4 indent, wrap in `()` if desired. You can also use `|` at the start of the line if you want each line to look the same (better for diffs, although I didn't set up a lot of my unions this way initially).